index.html

```diff
@@ -3262,6 +3306,36 @@ <h5>Existing implementations</h5>
 <dt>bing-maps-embed</dt>
 <dd><a>partial support</a>: does not support markers for a single location, but it is possible to display a map centered around a particular street address with no marker</dd>
 </dl>
+
+
+<h5>Uses beyond mapping</h5>
+<p>
+There are many cases where a website may expect an address from a user.
+An input type specifically for addresses has a wide variety applications other than displaying a map,
+such as reducing user error when facilitating e-commerce and delivery services.
+</p>
+
```

Comment:
We're missing a “Related Specifications” subhead in this section.

Some points to mention for it:

- the complex/overlapping set of address parts described for the [HTML `autocomplete` attribute](https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#autofilling-form-controls:-the-autocomplete-attribute)

- The [`PaymentAddress` interface](https://www.w3.org/TR/payment-request/#dom-paymentaddress) in the Payment Request API

These could be added in as a TODO for now:
```suggestion

<h5>Related web specifications</h5>
<p>
TODO: mention attempts to standardize what an “address” is,
in HTML autocomplete attribute
and Payment Request API.
</p>
```